code/game/objects/items/stacks/tiles/tile_iron.dm

```diff
@@ -5,7 +5,7 @@
 	icon_state = "tile"
 	inhand_icon_state = "tile"
 	force = 6
-	mats_per_unit = list(/datum/material/iron=SMALL_MATERIAL_AMOUNT*5)
+	mats_per_unit = list(/datum/material/plasma=SHEET_MATERIAL_AMOUNT*0.25)
```

Comment:
plasma?

Reply:
copypasted the code from plasma, forgot to change it, fixed in commit d3838d3